azure-sink-connector/src/integration-test/java/io/aiven/kafka/connect/azure/ParquetIntegrationTest.java

```diff
@@ -0,0 +1,316 @@
+                final var key = "key-" + cnt;
+                final String value;
+                final String payload;
+                if (i < 5) { // NOPMD literal
+                    payload = "{" + "\"name\": \"user-" + cnt + "\"}";
+                    value = String.format(jsonMessagePattern, jsonMessageSchema, payload);
+                } else {
+                    payload = "{" + "\"name\": \"user-" + cnt + "\", \"value\": \"value-" + cnt + "\"}";
+                    value = String.format(jsonMessagePattern, jsonMessageNewSchema, payload);
+                }
+                expectedRecords.add(String.format("{\"value\": %s}", payload));
+                sendFutures.add(sendMessageAsync(testTopic0, partition, key.getBytes(StandardCharsets.UTF_8),
+                        value.getBytes(StandardCharsets.UTF_8)));
+                cnt += 1;
+            }
+        }
+        getProducer().flush();
+        for (final Future<RecordMetadata> sendFuture : sendFutures) {
+            sendFuture.get();
+        }
+
+        final List<String> expectedBlobs = Arrays.asList(getBlobName(0, 0, compression), getBlobName(0, 5, compression),
+                getBlobName(1, 0, compression), getBlobName(1, 5, compression), getBlobName(2, 0, compression),
+                getBlobName(2, 5, compression), getBlobName(3, 0, compression), getBlobName(3, 5, compression));
+
+        awaitAllBlobsWritten(expectedBlobs.size());
+        assertIterableEquals(expectedBlobs, testBlobAccessor.getBlobNames(azurePrefix));
+
+        final var blobContents = new ArrayList<String>();
+        for (final String blobName : expectedBlobs) {
+            final var records = ParquetUtils.readRecords(tmpDir.resolve(Paths.get(blobName)),
+                    testBlobAccessor.readBytes(blobName));
+            blobContents.addAll(records.stream().map(GenericRecord::toString).collect(Collectors.toList()));
+        }
+        assertIterableEquals(expectedRecords.stream().sorted().collect(Collectors.toList()),
+                blobContents.stream().sorted().collect(Collectors.toList()));
+    }
+    private Map<String, String> basicConnectorConfig(final String compression) {
+        final Map<String, String> config = new HashMap<>();
+        config.put("name", CONNECTOR_NAME);
+        config.put("connector.class", AzureBlobSinkConnector.class.getName());
+        config.put("key.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
+        config.put("value.converter", "org.apache.kafka.connect.converters.ByteArrayConverter");
+        config.put("tasks.max", "1");
+        config.put("azure.storage.container.name", testContainerName);
+        if (useFakeAzure()) {
+            config.put("azure.storage.connection.string", azureEndpoint);
+        } else {
+            config.put("azure.storage.connection.string", azureConnectionString);
+        }
+
+        config.put("file.name.prefix", azurePrefix);
```

Comment:
also other configs like these can be defined as constants

Reply:
I wonder if it will be a good idea to reuse constants that are defined in `AzureBlobSinkConfig`?

Reply:
Yes, thats ok I think.

Reply:
Done